File: 06_Layouts/app/src/main/java/com/codewithharry/layouts/MainActivity.java
package com.codewithharry.layouts;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    public void start(View view)
    {
        Toast.makeText(this, "starting process", Toast.LENGTH_SHORT).show();
    }
    public void pause(View view)
    {
        Toast.makeText(this, "paused", Toast.LENGTH_SHORT).show();
    }
    public void stop(View view)
    {
        Toast.makeText(this, "stoping process", Toast.LENGTH_SHORT).show();
    }
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: 14_button/app/src/main/java/com/codingb/button/MainActivity.kt
package com.codingb.button

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun showtoast(view: View) {
        Toast.makeText(this , "this is start button click" , Toast.LENGTH_SHORT).show()
    }
}